README.md

```diff
@@ -68,24 +86,118 @@ The computation of the Shapley Values quickly becomes intensive when the number
     experiment_name: my_custom_experiment
     n_repeats: 10
     scenario_params_list:
-     - nodes_counts: 3
-       amounts_per_node: [0.4, 0.3, 0.3] 
+     - partners_count: 3
+       amounts_per_partner: [0.4, 0.3, 0.3] 
        samples_split_option: 'random'
-       aggregation_weighting: 'data-volume'
+       aggregation_weighting: 'data_volume'
        single_partner_test_mode: 'global'
        epoch_count: 38
        minibatch_count: 20
-     - nodes_counts: 4
-       amounts_per_node: [0.3, 0.3, 0.1, 0.3] 
+     - partners_count: 4
+       amounts_per_partner: [0.3, 0.3, 0.1, 0.3] 
        samples_split_option: 'stratified'
-       aggregation_weighting: 'data-volume'
+       aggregation_weighting: 'data_volume'
        single_partner_test_mode: 'global'
        epoch_count: 38
        minibatch_count: 20
     ```
 - Then execute `main.py -f config.yml`
 - A `results.csv` file will be generated in a new folder for your experiment under `/experiments`. You can read this raw `results.csv` file or use the `analyse_results.ipynb` notebook to quickly generate figures.
 
+### Config file parameters
+
+#### Experiment-level parameters
+
+An experiment regroups one or several scenarios to be run.
+
+`experiment_name`: `str`  
+How the experiment will be named in output files.  
+Example: `experiment_name: my_first_experiment`
+  
+`n_repeats`: `int`  
+How many times the experiment is run.  
+Example: `n_repeats: 2`
+
+#### Scenario-level parameters
+
+##### Definition of collaborative scenarios
+
+`partners_count`: `int`  
+Number of partners in the mocked collaborative ML scenario.  
+Example: `partners_count: 4`
+
+`amounts_per_partner`: `[float]`  
+Percentages of the original dataset each partner receives to mock a collaborative ML scenario where each partner provides data for the ML training.  
+Example: `amounts_per_partner: [0.3, 0.3, 0.1, 0.3]`
+
+`samples_split_option`: `random` or `stratified`  
+How the original dataset data samples are split among partners:
+
+- `random`: the dataset is shuffled and the split is done randomly, ensuring a homogeneous data distribution among partners
+- `stratified`: the dataset is stratified per labels, so that partners have datasets covering different regions of space
+
+`corrupted_partners`: `[not_corrupted (default), shuffled or corrupted]`  
```

Comment:
`corrupted_partners` makes me believe that one of the partner belongs to the mafia :).
Suggestions: corrupted_data ? data_corruption_status ? data_quality_per_partner ?

Reply:
Done, renamed as `corrupted_datasets`